cli/client/http.go

```diff
@@ -119,60 +131,56 @@ func HTTPQuery(request *http.Request) *http.Response {
 		switch err.(type) {
 		case x509.UnknownAuthorityError:
 			// custom suggestions for a certificate error:
-			log.Printf("HTTP %s Query for %s failed: %s", request.Method, request.URL, err)
-			log.Printf("- Is someone intercepting the connection to steal your credentials?")
-			log.Printf("- Is the cluster CA certificate configured correctly? Check 'dcos config show core.ssl_verify'.")
-			log.Fatalf("- To ignore the unvalidated certificate and force your command (INSECURE), use --force-insecure")
+			logMessage("HTTP %s Query for %s failed: %s", request.Method, request.URL, err)
+			logMessage("- Is someone intercepting the connection to steal your credentials?")
```

Comment:
Let's not log this. It's terrifying and will lead to users filing pointless support tickets.